src/module_PI/Arduino_Uno/CommandListener.cpp

```diff
@@ -54,5 +54,8 @@ void CommandListener::executeCommand(byte* commandBytes){
     case 30006:
       alarm->resetCalibration();
       break;
+    case 30007:
```

Comment:
Please add these commands to the [Logging Readme](https://github.com/Lastrellik/Calico-Home-Security/blob/master/docs/development/logging.md) so that they stay in sync. This applies to this one and the two others above that aren't in the readme either.